plugin.js

```diff
@@ -102,8 +105,10 @@ function casPlugin(server, options) {
         request.session.username = result.user;
         request.session.attributes = result.attributes || {};
 
-		// Save raw cas result for processing by client
-        request.session.raw_cas = result;
+        // Save raw cas result for processing by client
+        if (_options.value.saveRawCAS) {
+          request.session.raw_cas = result;
+        }
```

Comment:
I didn't notice this before, but let's stick to camel case. So `request.session.rawCas`. But, not that I think about it, what do you think of `request.session.casXml`?